Extension/background.js

```diff
@@ -2,6 +2,23 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+function copyToClipboard(text) {
+  var ta = document.getElementById('ta');
+  ta.style.display = 'block';
+  ta.value = text;
+  ta.select();
+  document.execCommand('copy');
+  ta.style.display = 'none';
+}
+
+function snagTitle(response){

```

Comment:
nit: outside of the original context (where it was specified as the object sent back by a callback), the name `response` isn't very informative. Perhaps something that conveys that it's information about an issue?